presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/PackageAPI.java

```diff
@@ -129,13 +128,13 @@ public synchronized Response uninstall(
             @QueryParam("ignoreErrors") @DefaultValue("false") @ApiParam("If true, warnings are ignored during uninstall") boolean ignoreErrors)
     {
         LOGGER.debug("DELETE /package checkDependencies: {} ; ignoreErrors: {}", checkDependencies, ignoreErrors);
-        new Thread(new PrestoAsynchronousCommand(new PrestoUninstaller(installedPackageType, checkDependencies, ignoreErrors))).start();
-        return Response.status(ACCEPTED).entity(text("Presto is being uninstalled")).build();
+        new Thread(new PrestoAsynchronousCommand(new PrestoUninstaller(PackageType.RPM, checkDependencies, ignoreErrors))).start();
+        return Response.status(ACCEPTED).entity("Presto is being uninstalled").build();
     }
 
     public enum PackageType
     {
-        RPM, TARBALL;
+        RPM, TARBALL
```

Comment:
You probably want this enum somewhere else, should be in some utility/ a class of its own. i.e enum class.